roles/masters/tasks/scripts.yml

```diff
@@ -15,6 +15,7 @@
        owner: root
        group: root
        mode: 0755
+       force: no
```

Comment:
I think we should overide this script in order to provide every update. And if the customer need a custom script, he have to fork on create a custom base on our upstream ?